### PR TITLE
Use new platform link

### PR DESCRIPTION
**Pull request by iotvietmember**
The VBLUno51 has platform link at https://os.mbed.com/platforms/VBLUNO51/

### CODE REVIEW

**Comment by iotvietmember** [comment]
Hi @c1728p9 , could you please review this commit soon? Thanks

File: source/board/vbluno51.c [hunk 1]
```diff
@@ -32,9 +32,9 @@ const char *board_id = "C006";
 __attribute__((aligned(4)))
 const vfs_filename_t daplink_url_name =       "VBLUNO51HTM";
 __attribute__((aligned(4)))
-const vfs_filename_t daplink_drive_name =     "DAPLINK    ";
+const vfs_filename_t daplink_drive_name =     "VBLUNO51   ";
```
Comment (c1728p9) [inline source/board/vbluno51.c]: This name may not be properly detected with mbed-ls. CC @theotherjimmy

Comment (theotherjimmy) [reply source/board/vbluno51.c]: Yep, probably would not be recognized by mbed-ls

Comment (iotvietmember) [reply source/board/vbluno51.c]: Do I have to use mbed.htm instead of vbluno51.htm or use DAPLINK for drive name?

Comment (c1728p9) [reply source/board/vbluno51.c]: DAPLINK or MBED should work

Comment (iotvietmember) [reply source/board/vbluno51.c]: OK. I will rebase this commit

**Comment by iotvietmember** [comment]
@c1728p9 @theotherjimmy  I have juse rebased the commit and fix the problem. Thanks

**Comment by iotvietmember** [comment]
@c1728p9 `/morph test` command seem be to do not run

**Pull request merged**